### ISSUE TITLE
Make country data in table & filters consistent

### ISSUE THREAD

**Issue by maryjom** [w3c/wai-policies-prototype#151]
Some data has WCAG 2.0 Level A while the rest simply say WCAG 2.0. Be consistent. Same with the type of law/policy.  Once consistent in the country data, remove extra filters.

**Comment by iamjolly** [comment]
The filters may remove themselves once those different entries are removed out of the country data files. I believe we're generating the filters dynamically from that data.

**Comment by yatil** [comment]
Yes. India seems to have WCAG 2.0 A – Actually, if it is in the table it should be in the search as well.

I think we want to close this :-)
